[2025-05-28T03:04:13.334+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-28T03:04:13.347+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_2.wait_for_dag1 scheduled__2025-05-27T00:00:00+00:00 [queued]>
[2025-05-28T03:04:13.354+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_2.wait_for_dag1 scheduled__2025-05-27T00:00:00+00:00 [queued]>
[2025-05-28T03:04:13.354+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-05-28T03:04:13.365+0000] {taskinstance.py:2888} INFO - Executing <Task(ExternalTaskSensor): wait_for_dag1> on 2025-05-27 00:00:00+00:00
[2025-05-28T03:04:13.373+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3023) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-28T03:04:13.374+0000] {standard_task_runner.py:72} INFO - Started process 3024 to run task
[2025-05-28T03:04:13.373+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_2', 'wait_for_dag1', 'scheduled__2025-05-27T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/Sesion2/Ejemplo5.py', '--cfg-path', '/tmp/tmpn2p1rsz2']
[2025-05-28T03:04:13.375+0000] {standard_task_runner.py:105} INFO - Job 35: Subtask wait_for_dag1
[2025-05-28T03:04:13.386+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-28T03:04:13.411+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_2.wait_for_dag1 scheduled__2025-05-27T00:00:00+00:00 [running]> on host 31d54648d280
[2025-05-28T03:04:13.412+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-28T03:04:13.474+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_2' AIRFLOW_CTX_TASK_ID='wait_for_dag1' AIRFLOW_CTX_EXECUTION_DATE='2025-05-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-27T00:00:00+00:00'
[2025-05-28T03:04:13.475+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-28T03:04:13.490+0000] {baseoperator.py:405} WARNING - ExternalTaskSensor.execute cannot be called outside TaskInstance!
[2025-05-28T03:04:13.491+0000] {external_task.py:258} INFO - Poking for tasks ['end'] in dag dag_1 on 2025-05-27T00:00:00+00:00 ... 
[2025-05-28T03:04:13.492+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/session.py:97 DeprecationWarning: This method is deprecated and will be removed in future.
[2025-05-28T03:05:13.498+0000] {external_task.py:258} INFO - Poking for tasks ['end'] in dag dag_1 on 2025-05-27T00:00:00+00:00 ... 
[2025-05-28T03:06:13.505+0000] {external_task.py:258} INFO - Poking for tasks ['end'] in dag dag_1 on 2025-05-27T00:00:00+00:00 ... 
[2025-05-28T03:06:13.514+0000] {base.py:336} INFO - Success criteria met. Exiting.
[2025-05-28T03:06:13.521+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-28T03:06:13.521+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_2, task_id=wait_for_dag1, run_id=scheduled__2025-05-27T00:00:00+00:00, execution_date=20250527T000000, start_date=20250528T030413, end_date=20250528T030613
[2025-05-28T03:06:13.557+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-28T03:06:13.579+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-28T03:06:13.580+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
